invenio_app_ils/records_relations/views.py

```diff
@@ -214,6 +213,94 @@ def _delete_sibling_relation(self, record, relation_type, payload):
         )
         return modified_record, record, second
 
+    def _validate_sequence_creation_payload(self, record, payload):
+        """Validate the payload when creating a new sequence relation."""
+        try:
+            next_pid = payload.pop("next_pid")
+            next_pid_type = payload.pop("next_pid_type")
+            previous_pid = payload.pop("previous_pid")
+            previous_pid_type = payload.pop("previous_pid_type")
+        except KeyError as key:
+            raise RecordRelationsError(
+                "The `{}` is a required field".format(key)
+            )
+
+        if record["pid"] != next_pid and record["pid"] != previous_pid:
+            raise RecordRelationsError(
+                "Cannot create a relation for other record than one with PID "
+                "`{}`".format(record["pid"])
+            )
+
+        if next_pid == previous_pid:
+            raise RecordRelationsError(
+                "Cannot create a sequence with the same next PID `{}`"
+                "and previous PID `{}`".format(next_pid, previous_pid)
+            )
+
+        return next_pid, next_pid_type, previous_pid, previous_pid_type, \
+            payload
+
+    def _create_sequence_relation(self, record, relation_type, payload):
+        """Create a Sequence relation.
+
+        Expected payload:
+
+            {
+                next_pid: <pid_value>,
+                next_pid_type: <pid_type>,
+                previous_pid: <pid_value>,
```

Comment:
from the conversation I remember we create only the `-> next` relation, no? can we also create the previous?

Reply:
We had a couple of talks with @ntarocco about how we want to create the relations. My initial approach was to be explicitly set `next` in one request and `previous` on another, but we agreed to keep it simple and create the two way connection at once, which is the most common (if not every) case.